Dapper.Contrib/SqlMapperExtensions.cs

```diff
@@ -314,7 +314,7 @@ private static string GetTableName(Type type)
                 isList = true;
                 type = type.GetElementType();
             }
-            else if (type.IsGenericType())
+            else if (type.IsGenericType() && type.GetTypeInfo().ImplementedInterfaces.Any(x => x.Name == "IEnumerable`1"))

```

Comment:
Any reason to not use `.Any(ti => ti.IsGenericType && ti.GetGenericTypeDefinition() == typeof(IEnumerable<>))` for these checks?

Reply:
@NickCraver No. That's a better way of doing it. I've changed it.

`ti.IsGenericType` does not work in .NET Standard 1.3, but
`ti.IsGenericType()` does, so instead of
`.Any(ti => ti.IsGenericType && ti.GetGenericTypeDefinition() == typeof(IEnumerable<>))`
this needs to be
`.Any(ti => ti.IsGenericType() && ti.GetGenericTypeDefinition() == typeof(IEnumerable<>))`.